make_report.py

```diff
@@ -30,7 +30,27 @@ def build(self) -> List[Section]:
         return self._sections
 
 
-def get_image_caption(filename: str) -> str:
+def read_image_captions(input_photo_dir: str) -> Dict[str, str]:
```

Comment:
If we're always using Python 3.9+ (which I think we can say we are) this can just be `dict[str, str]` and don't need to specially import `Dict`